netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java

```diff
@@ -329,7 +376,7 @@ public Handler newHandler(GrpcHttp2ConnectionHandler handler) {
       HttpClientCodec httpClientCodec = new HttpClientCodec();
       final HttpClientUpgradeHandler upgrader =
           new HttpClientUpgradeHandler(httpClientCodec, upgradeCodec, 1000);
-      return new BufferingHttp2UpgradeHandler(upgrader);
+      return new BufferingHttp2UpgradeHandler(httpClientCodec, upgrader);
```

Comment:
So client-side Upgrade support was previously broken? That's what it looks like, but wanted to confirm.